api.md

```diff
@@ -0,0 +1,171 @@
+## API
+
+All requests can be authenticated by a signed cookie with the session ID.
+
+### POST /session
+Start a new observation session.
+
+Request:
+```
+{
+  name: 'name',
+  methodology: {
+    observation_type: 'focal_animal',
+    focal_animal: 'tarzan',
+    interval_seconds: '100'
+  }
+
+  note: 'thing',
+
+  // extra metadata
+  location: {
+    lat, long
+  }
+}
+```
+
+Response:
+
+```
+Headers:
+X-Session-Id=YourResourceKey
+
+or
+
+{
+  session: {
+    id: your_resource_key
+  }
+}
+```
+
+### PATCH /session
+Start or finish the session.
+
+```
+{
+  session: {
+    state: 'finished' // (start|finish)
+  }
+}
+```
+
+### GET /:user_id/schemes
```

Comment:
Let's do `/schemes` instead, since we will already know who the logged in user is.